2996_reflection/reflection.md

```diff
@@ -4305,27 +4403,79 @@ namespace std::meta {
 :::
 :::
 
-### [meta.reflection.names] Reflection names and locations {-}
+### [meta.reflection.operators] Operator representations
 
 ::: std
 ::: addu
-```cpp
-consteval string_view name_of(info r);
-consteval u8string_view u8name_of(info r);
-```
 
-[#]{.pnum} *Constant When*: If returning `string_view`, the unqualified name is representable using the ordinary string literal encoding. `r` represents a declared entity.
+[#]{.pnum} This enum class specifies constants used to identify operators that can be overloaded, with the meanings listed in Table 1. The values of the constants are distinct.
+
+<center> Table 1: Enum class `operators` [meta.reflection.operators]</center>
+<table  style="text-align:center">
+<tr><th>[Constant]{.addu}</th><th>[Corresponding operator]{.addu}</th></tr>
+<tr><td>[`op_new`]{.addu}</td><td>[`operator new`]{.addu}</td></tr>
+<tr><td>[`op_delete`]{.addu}</td><td>[`operator delete`]{.addu}</td></tr>
+<tr><td>[`op_array_new`]{.addu}</td><td>[`operator new[]`]{.addu}</td></tr>
+<tr><td>[`op_array_delete`]{.addu}</td><td>[`operator delete[]`]{.addu}</td></tr>
+<tr><td>[`op_co_await`]{.addu}</td><td>[`operator coawait`]{.addu}</td></tr>
+<tr><td>[`op_parentheses`]{.addu}</td><td>[`operator()`]{.addu}</td></tr>
+<tr><td>[`op_square_brackets`]{.addu}</td><td>[`operator[]`]{.addu}</td></tr>
+<tr><td>[`op_arrow`]{.addu}</td><td>[`operator->`]{.addu}</td></tr>
+<tr><td>[`op_arrow_asterisk`]{.addu}</td><td>[`operator->*`]{.addu}</td></tr>
+<tr><td>[`op_tilde`]{.addu}</td><td>[`operator~`]{.addu}</td></tr>
+<tr><td>[`op_exclamation_mark`]{.addu}</td><td>[`operator!`]{.addu}</td></tr>
+<tr><td>[`op_plus`]{.addu}</td><td>[`operator+`]{.addu}</td></tr>
+<tr><td>[`op_minus`]{.addu}</td><td>[`operator-`]{.addu}</td></tr>
+<tr><td>[`op_asterisk`]{.addu}</td><td>[`operator*`]{.addu}</td></tr>
```

Comment:
Can't wait to have discussions about these names. Like should this be `op_star`, etc.

Reply:
Gonna be fucking brutal.